README.md

```diff
@@ -100,7 +114,7 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+Schema is the set of attributes in the table for a model/entity. In a one-to-many relationship using the people and wishes example, a person has many wishes, and wishes belong to a person. The wishes table will thus have a foreign key that belongs to the person.  
```

Comment:
:+1: 
This is correct, but I'd like to see an example of what the foreign key looks like, e.g. `person_id`.